src/backupNg/backupNg.spec.js

```diff
@@ -30,5 +30,46 @@ describe("backupNg", () => {
       expect(omit(backupNg, "id", "userId")).toMatchSnapshot();
       expect(backupNg.userId).toBe(xo._user.id);
     });
+
+    it("creates a new backup job with schedules", async () => {
+      let scheduleId = "scheduleIdTemp";
+      const { id: jobId } = await xo.createTempBackupNgJob({
+        ...defaultBackupNg,
+        schedules: {
+          [scheduleId]: {
+            name: "scheduleTest",
+            cron: "0 * * * * *",
+          },
+        },
+        settings: {
+          ...defaultBackupNg.settings,
+          [scheduleId]: { snapshotRetention: 1 },
+        },
+      });
+
+      const backupNgJob = await xo.call("backupNg.getJob", { id: jobId });
+      for (const elt in backupNgJob.settings) {
```

Comment:
`backupNgJob` settings will be browse two times. One for getting schedule id and one for the test. Maybe we could do the test in this loop. Also rename `elt`  ->  `scheduleId`.

Reply:
OK :+1: 

But if I do the test in this loop, I think I could not rename `elt` in `scheduleId`, because I should test the other property. What do you think ? 

(In addition, the global variable is already named `scheduleId`)

Reply:
Right, you can let it element, because it can not be an id.

Reply:
Also in this test do we need to test also the `reportWhen` ?

Reply:
I think we don't need to test the `reportWhen`. I did this test in order to test all the elements in the object `defaultBackupNg` but I can remove this one, it is not the goal of this test :+1:  .

Reply:
`key`?

Reply:
OK :+1: